repos/system_upgrade/el7toel8/actors/scanmemory/tests/test_scanmemory.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+from leapp.libraries.actor import library
+from leapp.libraries.common import testutils
+from leapp.libraries.stdlib import api
+from leapp.models import MemoryInfo
+
+
+class mocked_get_memoryinfo(object):
+    def __init__(self, filename):
+        self.filename = filename
+
+    def __call__(self):
+        """
+        Return lines of the self.filename test file located in the files directory.
+
+        Those files contain /proc/meminfo content from machine.
+        """
+        return dict((i.split()[0].rstrip(':'), int(i.split()[1])) for i in open(os.path.join('tests/files', self.filename), 'rb').readlines())  # noqa: E501; pylint: disable=line-too-long
```

Comment:
Please see comment above. 

Another suggestion will be to move code to a library and reuse in both tests.

Reply:
fixed.